olive/workflows/run/config.py

```diff
@@ -55,6 +55,7 @@ class RunConfig(ConfigBase):
     azureml_client: AzureMLClientConfig = None
     input_model: ModelConfig
     systems: Dict[str, SystemConfig] = None
+    data_root: str = None
```

Comment:
Can you also update the documentation about data_root config and how it is related to data_dir for each pass and evaluator?

Reply:
sure, will do